lib/spree/backend/engine.rb

```diff
@@ -27,6 +27,9 @@ class Engine < ::Rails::Engine
       config.after_initialize do
         Rails.application.reload_routes!
         Rails.application.config.spree_backend.main_menu = Spree::Admin::MainMenu::DefaultConfigurationBuilder.new.build
+        Rails.application.config.spree_backend.order_tabs = Spree::Admin::Resources::OrderDefaultTabsBuilder.new.build
```

Comment:
Maybe instead of adding three keys (and potentially more, as I think "Store", "Taxon" entities also have tabs ;) ), we could introduce an intermediate structure?
So that you'd have `Rails.application.config.spree_backend.tabs.order` etc. ?

Reply:
I've updated it but I was having troubles with nesting a struct. I'd appreciate if you could throw a glance. Maybe you'd have a hint on some more elegant solution. 🙂